src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -270,9 +270,6 @@
       BeforeTargets="CoreCompile"
       Inputs="@(_SharedRuntimeAssemblies)"
       Outputs="$(_GeneratedProfileClass)">
-    <ItemGroup>
-      <_SharedRuntimeAssemblies Remove="$(_SharedRuntimeBuildPath)v1.0\FSharp.Core.dll" />
-    </ItemGroup>
```

Comment:
I checked `src\Xamarin.Android.Build.Tasks\obj\Debug\Profile.g.cs` and `FSharp.Core.dll` isn't being written to it at all, anyway:

```csharp
∩╗┐using System.Collections.Generic;

namespace Xamarin.Android.Tasks {
	public partial class Profile {
		// KEEP THIS SORTED ALPHABETICALLY, CASE-INSENSITIVE
		public static readonly string [] SharedRuntimeAssemblies = new []{
			"I18N.CJK.dll",
			"I18N.dll",
			"I18N.MidEast.dll",
			"I18N.Other.dll",
			"I18N.Rare.dll",
			"I18N.West.dll",
			"Java.Interop.dll",
			"Microsoft.CSharp.dll",
			"Mono.Android.dll",
			"Mono.Android.Export.dll",
			"Mono.Btls.Interface.dll",
			"Mono.CompilerServices.SymbolWriter.dll",
			"Mono.CSharp.dll",
			"Mono.Data.Sqlite.dll",
			"Mono.Data.Tds.dll",
			"Mono.Posix.dll",
			"Mono.Security.dll",
			"mscorlib.dll",
			"OpenTK.dll",
			"OpenTK-1.0.dll",
			"System.ComponentModel.Composition.dll",
			"System.ComponentModel.DataAnnotations.dll",
			"System.Core.dll",
			"System.Data.DataSetExtensions.dll",
			"System.Data.dll",
			"System.Data.Services.Client.dll",
			"System.dll",
			"System.EnterpriseServices.dll",
			"System.IdentityModel.dll",
			"System.IO.Compression.dll",
			"System.IO.Compression.FileSystem.dll",
			"System.Json.dll",
			"System.Net.dll",
			"System.Net.Http.dll",
			"System.Net.Http.WinHttpHandler.dll",
			"System.Numerics.dll",
			"System.Numerics.Vectors.dll",
			"System.Reflection.Context.dll",
			"System.Runtime.CompilerServices.Unsafe.dll",
			"System.Runtime.Serialization.dll",
			"System.Security.dll",
			"System.ServiceModel.dll",
			"System.ServiceModel.Internals.dll",
			"System.ServiceModel.Web.dll",
			"System.Transactions.dll",
			"System.Web.Services.dll",
			"System.Windows.dll",
			"System.Xml.dll",
			"System.Xml.Linq.dll",
			"System.Xml.Serialization.dll",
		};
	}
}
```